deploy/kustomize/patches/worker-hcs.yaml

```diff
@@ -83,16 +83,6 @@
           value: ${WORKER_HCS_WORKER_QUEUE}
         - name: WORKER_PROC_ALIVE_TIMEOUT
           value: ${WORKER_PROC_ALIVE_TIMEOUT}
-        - name: SCHEDULE_CHECK_INTERVAL
-          value: ${SCHEDULE_CHECK_INTERVAL}
-        - name: REMOVE_EXPIRED_REPORT_DATA_ON_DAY
```

Comment:
Possible it's my eyes. How are we handling the schedule for the expired data task now? Or is that only the scheduler need actually care about this? And it's not needed in the workers? Am I answering my own question?

Reply:
Yep, you're answering your own question. Only the scheduler requires this env variable.